{{cookiecutter.repo_name}}/.pre-commit-config.yaml

```diff
@@ -17,7 +17,7 @@ repos:
     exclude: docs/|setup.cfg
 
 - repo: https://github.com/myint/docformatter
```

Comment:
The given url (https://github.com/myint/docformatter) appears to redirect to https://github.com/PyCQA/docformatter which is the correct one, I believe. While the redirection appears to be working, perhaps for clarity this url should be updated?

Reply:
Thanks Chris, great spot 👍